samples/excel/85-preview-apis/range-dependents.yaml

```diff
@@ -0,0 +1,176 @@
+        async function setup() {
+          await Excel.run(async (context) => {
+            context.workbook.worksheets.getItemOrNullObject("Sample2019Data").delete();
+            context.workbook.worksheets.getItemOrNullObject("Sample2020Data").delete();
+
+            // Set up the first sample table.
+            const sheet1 = context.workbook.worksheets.add("Sample2019Data");
+            const data1 = [
+              ["Product", "Qty", "Unit Price", "Total Price 2019"],
+              ["Almonds", 2, 7.5, "=C3 * D3"],
+              ["Coffee", 1, 34.5, "=C4 * D4"],
+              ["Chocolate", 5, 9.56, "=C5 * D5"]
+            ];
+
+            const range1 = sheet1.getRange("B2:E5");
+            range1.values = data1;
+            range1.format.autofitColumns();
+
+            // Set up the second sample table.
+            const sheet2 = context.workbook.worksheets.add("Sample2020Data");
+            const data2 = [
+              ["Product", "Qty", "Unit Price", "Total Price 2020", "Average Annual Price"],
+              ["Almonds", 2, 8.0, "=C3 * D3", "=Sample2019Data!E3 + E3 / 2"],
+              ["Coffee", 1, 36.5, "=C4 * D4", "=Sample2019Data!E4 + E4 / 2"],
+              ["Chocolate", 5, 11.2, "=C5 * D5", "=Sample2019Data!E5 + E5 / 2"]
+            ];
+
+            const range2 = sheet2.getRange("B2:F5");
+            range2.values = data2;
+            range2.format.autofitColumns();
+
+            // Style the tables.
+            const header1 = range1.getRow(0);
+            header1.format.fill.color = "#4472C4";
+            header1.format.font.color = "white";
+
+            const header2 = range2.getRow(0);
+            header2.format.fill.color = "#4472C4";
+            header2.format.font.color = "white";
+
+            sheet2.activate();
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to find and highlight the dependents of the currently selected cell. </p>
+          <p>Dependent cells contain formulas that refer to other cells.</p>
+        </section>
+        <section class="setup ms-font-m">
+          <h3>Set up</h3> 
+          <button id="setup" class="ms-Button"> 
+            <span class="ms-Button-label">Add sample data</span> 
+          </button>
+        </section>
+        <section class="samples ms-font-m">
+          <h3>Try it out</h3>
+          <button id="select-D3" class="ms-Button">
+            <span class="ms-Button-label">Select a cell with dependents across worksheets</span>
```

Comment:
Since the button text tends to wrap awkwardly in Script Lab, consider changing the button text to "Select D3" and adding a label saying "D3 has dependents across worksheets.", or something like that.